packages/build/tests/constants/fixtures/functions_src_dynamic/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "test",
+  "type": "module",
+  "description": "test",
+  "license": "MIT",
+  "repository": "test",
+  "dependencies": {}
+}
```

Comment:
Some of the test fixtures now need a `package.json`.
One of the main reasons are test fixture directories which are copied to the global temporary. If the files in the fixture directory is using ES modules, a `package.json` is then needed to specify `"type": "module"`.